README.rst

```diff
@@ -1,4 +1,4 @@
-.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+.. image:: http://i.imgur.com/gOHfIWL.png
```

Comment:
I think you should create a pull at assets repo, and then use that URL, as used earlier.